luthier/config/src/utils/tableUtils.lua

```diff
@@ -12,33 +12,72 @@ local misc = require 'utils./misc'
 -- Build Config
 -------------------------------------------------------------------------------
 
+--- Build a specific physical table into a json object.
```

Comment:
This is incorrect. This function doesn't actually build a JSON object, but
rather another Lua table. It massages the user provided physical table
configuration into a form that makes it easier for Fili to consume.

Reply:
Changed wording